app/src/pages/project/SpanFilterConditionField.tsx

```diff
@@ -396,6 +408,18 @@ function FilterConditionBuilder(props: {
           initialSnippet="evals['Hallucination'].score < 1"
           onAddFilterConditionSnippet={onAddFilterConditionSnippet}
         />
+        <FilterConditionSnippet
+          key="metadata"
+          label="filter by metadata"
+          initialSnippet="metadata['topic'] == 'dashboard'"
+          onAddFilterConditionSnippet={onAddFilterConditionSnippet}
+        />
+        <FilterConditionSnippet
+          key="substring"
+          label="filter by substring"
+          initialSnippet="'gpt' in llm.model_name"
```

Comment:
```suggestion
          initialSnippet="'gpt' in input.value"
```
probably a more common use-case